lib/birthday_reader.rb

```diff
@@ -1,49 +1,49 @@
-class Birthday
+class BirthdayReader
   @@filepath = nil
 
   def self.filepath=(path=nil)
     @@filepath = File.join(APP_ROOT, path)
   end
 
   #checks if the file we are looking exists in the correct directory
-  def self.file_exists?
-    if @@filepath && File.exists?(@@filepath)
-      return true
-    else
-      return false
-    end
+  def self.file_exist?
+    return (@@filepath and File.exist?(@@filepath))
   end
 
   #checks if the file is valid
   def self.file_usable?
     return false unless @@filepath
-    return false unless File.exists?(@@filepath)
+    return false unless File.exist?(@@filepath)
     return false unless File.readable?(@@filepath)
     return true
   end
 
   #if file doesnt exist it creates one
   def self.create_file
-    File.open(@@filepath, 'w') unless file_exists?
+    File.open(@@filepath, 'w') unless file_exist?
     return file_usable?
   end
 
-  #gets the array of that line in predefined order(1st name, last name, year, month, day)
+  #gets the array of that line in predefined order (1st name, last name, year, month, day)
   def import_line(line)
     line_array = line.split(' ')
-    @primeiro_nome, @ultimo_nome, @ano, @mes, @dia = line_array
+    @first_name, @last_name, @year, @month, @day = line_array
     return line_array
   end
 
   #creates the final array that will be worked with
   def self.get_birthdays
     birthdays = []
     if file_usable?
+      print 'Parsing names and birthdays (count='
       file = File.new(@@filepath, 'r')
       file.each_line do |line|
-        birthdays << Birthday.new.import_line(line.chomp)
+        birthdays << BirthdayReader.new.import_line(line.chomp)
```

Comment:
I think we can remove the `#import_line` function.

Instead we could write it like this:

``` ruby
birthdays << line.chomp.split
```